sympy/physics/control/control_plots.py

```diff
@@ -686,6 +686,8 @@ def bode_magnitude_numerical_data(system, initial_exp=-5, final_exp=5, **kwargs)
         The initial exponent of 10 of the semilog plot. Defaults to -5.
     final_exp : Number, optional
         The final exponent of 10 of the semilog plot. Defaults to 5.
+    freq_unit : string, optional
+        User can choose between 'rad/sec' (radians/second) and 'Hz' (Hertz) as frequency units.
```

Comment:
```suggestion
        User can choose between ``'rad/sec'`` (radians/second) and ``'Hz'`` (Hertz) as frequency units.
```
(Just at I suggested last time.)